[{"items": [{"tags": ["python", "windows", "tensorflow", "anaconda"], "owner": {"user_type": "does_not_exist", "display_name": "user8036523"}, "is_answered": true, "view_count": 5656, "accepted_answer_id": 45317751, "answer_count": 2, "score": 2, "last_activity_date": 1614273214, "creation_date": 1501044647, "last_edit_date": 1501049747, "question_id": 45317706, "content_license": "CC BY-SA 3.0", "link": "https://stackoverflow.com/questions/45317706/attributeerror-module-tensorflow-has-no-attribute-variable-error", "title": "AttributeError: module &#39;tensorflow&#39; has no attribute &#39;Variable&#39; &quot;ERROR&quot;", "body": "<p>I have used tensor-flow for ONE day, but there come some troubles, when I import tensor-flow, there would be AttributeError: 'module' object has no attribute 'variable'</p>\n\n<p>I use Windows10, Python 3.5.3, Anaconda-3 4.4.0</p>\n\n<p>here is my test code:</p>\n\n<pre><code>import tensorflow as tf\nmy_var = tf.Variable(tf.linspace(10.0, 13.0, 4)) \nwith tf.Session() as sess:\n    print (sess.run(my_var))\n</code></pre>\n\n<p>I got this error:\n<img src=\"https://i.stack.imgur.com/5WGSZ.png\" alt=\"Error\"></p>\n"}], "has_more": false, "quota_max": 300, "quota_remaining": 101}]